### DISCUSSION TITLE
Players Positions GUI/overlay

### DISCUSSION THREAD

**Discussion by nuexzz** [emoose/OutRun2006Tweaks#216] [Suggestions]
### Describe the feature you'd like to see added

Since this new GUI/overlay is really powerful, it would be great to see the positions of the players in real time without having to wait for the end of the race to know who is in the first places.
for example Daytona Usa psn shows it.
![online](https://github.com/user-attachments/assets/2d72a79b-28ac-4e78-abc1-2c640b710a0b)

I think the game's graphics window has enough space for it.
![online2](https://github.com/user-attachments/assets/5b8e6828-6b90-4167-be3b-f91558d65627)

**Comment by nuexzz** [comment]
I understand that if you are willing to add this Feature Request you will need directions.

``status = the online game when it starts to show everyone's positions and hide when it ends``

``how many players are in the session = to create the fields in GUI/overlay``

``player name = to write it in the field corresponding to the position``

``car color = to show the nickname of the same color``

``real position of the car online = to sort it in the corresponding field in GUI/overlay``


If it will make your job easier, i can would search for these addresses and paste them here..

For example here we have.

``status the online game when it starts to show everyone's positions and hide when it ends = 007D3A01 1byte``

``how many players are in the session 0080FB04 4bytes= 1(2players) 2(3players) 3(4players) 4(5players) 5(6players)``

``player name online 007D6978 (10string) =xxxxxxxxxx``

``chosen car color online
OR2006C2C.exe+3D69F0 4bytes = red1 yellow2 black3 white4 graysilver5 green6 light_blue7 dark_blue8
``

``real position of the car online = 007D6960 2bytes``

these addresses are from CE but obviously you can edit the values ​​to find them quickly with your debugger and use the correct addresses.

**Comment by emoose** [comment]
Neat idea, maybe we could use the games text drawing funcs as well so it'd use game fonts for it (like how cdswitcher draws track names)
Thanks for the addrs too, might try something with it soon

**Comment by nuexzz** [comment]
so I started working on it. I'll release it soon, it works with OutRun2006Tweak.

https://www.youtube.com/watch?v=eIp7dAfWOGk
